aip/general/0180.md

```diff
@@ -140,7 +149,9 @@ this guidance could ostensibly prevent _any_ change (which is not the intent).
 
 ## Changelog
 
-- **2022-06-??**: Added more links to other AIPs with compatibility concerns
+- **2022-08-11**: Clarified that moving components to a different file is
+  breaking.
```

Comment:
I think this isn't just clarification. You have added an entire section so stating:

Added "Moving components between files" section.

would be a lot more descriptive about your change.

Reply:
Done!